packages/cmd/src/services/convert/utils.ts

```diff
@@ -26,18 +29,20 @@ export function getTestCase(
       suiteName
     ),
     title: getTestTitle(testCase.name, suiteName),
-    state: hasFailure ? 'failed' : 'passed',
+    state: state,
     isFlaky: getTestFlakiness(),
-    expectedStatus: hasFailure ? 'skipped' : 'passed',
+    expectedStatus: 'passed',
     timeout: getTimeout(),
     location: getTestCaseLocation(suite?.file ?? ''),
     retries: getTestRetries(failures),
-    attempts: getTestAttempts(
-      testCase,
-      failures,
-      getISODateValue(suiteTimestamp),
-      time
-    ),
+    attempts: skipped
+      ? []
```

Comment:
@miguelangaranocurrents it looks like we need to put in a skipped attempt here (which we should add handling to getTestAttemps to do). Otherwise our dashboard UI for test details is a bit broken (and throws errors on loading stdout) If we have no attempt at all.

The attempt `status` should be `skipped` and the `_s` should be `pending`, to match what playwright ends up setting.